File: android/src/main/kotlin/com/example/design_engine/layer2_core/dark/EOnErrorDark.kt
package com.example.design_engine.layer2_core.dark
import com.example.design_engine.R
import com.example.design_engine.layer3_logic.AppTheme
object EOnErrorDark {
    fun getResId(activeTheme: AppTheme): Int {
        return when (activeTheme) {
            AppTheme.SYSTEM -> R.string.csyd42
            AppTheme.STANDARD -> R.color.cstd42
            AppTheme.BLUE -> R.color.cbd42
            AppTheme.GREEN -> R.color.cgd42
            AppTheme.RED -> R.color.crd42
            AppTheme.NOTHING -> R.color.cnod42
        }
    }
}

File: lib/layer2_core/dark/e_on_error_dark.dart
import '../../layer1_raw/design_token.dart';
import '../../layer1_raw/dark/system.dart';
import '../../layer1_raw/dark/standard.dart';
import '../../layer1_raw/dark/blue.dart';
import '../../layer1_raw/dark/green.dart';
import '../../layer1_raw/dark/red.dart';
import '../../layer1_raw/dark/nothing.dart';

class EOnErrorDark {
  static DesignToken getToken(dynamic activeTheme) {
    switch (activeTheme.toString().split('.').last) {
      case 'system': return csyd42;
      case 'standard': return cstd42;
      case 'blue': return cbd42;
      case 'green': return cgd42;
      case 'red': return crd42;
      case 'nothing': return cnod42;
      default: return cstd42;
    }
  }
}
